packages/react-ai/src/components/AIConversation/views/default/__tests__/Form.spec.tsx

```diff
@@ -57,4 +64,57 @@ describe('Form', () => {
     expect(fileInput.files).not.toBeNull();
     expect(fileInput.files![0]).toStrictEqual(testFile);
   });
+
+  it('updates input value through multiple change events with IME characters', async () => {
+    const result = render(<Form {...defaultProps} />);
+    expect(result.container).toBeDefined();
+
+    const textInput: HTMLInputElement = screen.getByTestId('text-input');
+
+    const preMatureInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '你',
+    });
+
+    const completeInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '你好',
+    });
+
+    await waitFor(() => fireEvent.change(textInput, preMatureInput));
+
+    await waitFor(() => fireEvent.change(textInput, completeInput));
+
+    expect(textInput.value).not.toEqual('');
+    expect(textInput.value).toBe('你好');
+  });
+
+  it('updates input value correctly after IME composition successfully ends', async () => {
+    const result = render(<Form {...defaultProps} />);
+    expect(result.container).toBeDefined();
+
+    const textInput: HTMLInputElement = screen.getByTestId('text-input');
+
+    const preMatureInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: 'しあわせ',
+    });
+
+    const completeInput = Object.defineProperty(textInput, 'value', {
+      configurable: true,
+      value: '幸せならおkです',
+    });
```

Comment:
Bit confused on why we are directly mutating the test input `value` property?

Reply:
Sorry for it being confusing, seems Hui also be concerned about this part. my understanding is that `value` property of `textInput` will not be updated in the actual `textInput` until they're actually passed with `.change` or `onCompositionEnd` which we are testing. Maybe do you think it could be more clear if I add an `expect` line on `textInput.value` before first `change()` is called to ensure the actual object's value is empty after the definition of `preMatureInput` and `completeInput`?